kotlin/services/kendra/Readme.md

```diff
@@ -0,0 +1,66 @@
+- [Deleting an Amazon Kendra index](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/example_code/kendra/src/main/kotlin/com/example/kendra/DeleteIndex.kt) (DeleteIndex command)
+- [Getting statistics about synchronizing Amazon Kendra with a data source](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/example_code/dynamodb/src/main/kotlin/com/example/dynamodb/ListDataSourceSyncJobs.kt) (ListDataSourceSyncJobs command)
+- [Querying an Amazon Kendra index](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/example_code/dynamodb/src/main/kotlin/com/example/dynamodb/QueryIndex.kt) (Query command)
+
+### Scenario
+
+- [Creating an Amazon Kendra index and data source, and syncing the data source](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/example_code/kendra/src/main/kotlin/com/example/kendra/CreateIndexAndDataSourceExample.kt) (Multiple commands)
+
+## Running the examples
+To run these examples, set up your development environment to use Gradle. For more information, 
+see [Setting up the AWS SDK for Kotlin](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/setup.html). 
+
+Some of these examples perform *destructive* operations on AWS resources, such as deleting an index. **Be very careful** when running an operation that deletes or modifies AWS resources in your account.
+
+## Tests
+⚠️ Running the tests might result in charges to your AWS account.
+
+You can test the Kotlin code examples for Amazon Kendra by running a test file named **KendraTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+
+You can run the JUnit tests from an IDE, such as IntelliJ, or from the command line by using Maven. As each test is run, you can view messages that inform you if the various tests succeed or fail. For example, the following message informs you that Test 3 passed.
+
+	Test 3 passed
+
+**WARNING**: _Running these JUnit tests manipulates real AWS resources and might incur charges on your account.
+
+ ### Properties file
+Before running the Amazon DynamoDB JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to run the JUnit tests. For example, you define an index name used for various tests. If you do not define all values, the JUnit tests fail.
+
+Define these values to successfully run the JUnit tests:
+
+- **indexName** - The name for the new index.
+- **dataSourceName** - The name for the new data source.
+- **indexDescription** - The description for the index. 
+- **indexRoleArn** – An Identity and Access Management (IAM) role that gives Amazon Kendra permissions to access your Amazon CloudWatch logs and metrics.
```

Comment:
Identity and Access Management (IAM) --> AWS Identity and Access Management (IAM)

Reply:
fixed